cmake/onnxruntime_webassembly.cmake

```diff
@@ -221,6 +221,7 @@ else()
     "SHELL:-s EXPORT_ALL=0"
     "SHELL:-s VERBOSE=0"
     "SHELL:-s FILESYSTEM=0"
+    "SHELL:-s INCOMING_MODULE_JS_API=[locateFile]"
```

Comment:
What's this?

Reply:
3.1.46 wants the module attributes listed explicit:
https://emscripten.org/docs/api_reference/module.html#compilation-settings